PlaceRouteHierFlow/PnRDB/datatype.h

```diff
@@ -32,6 +32,8 @@ struct MatchBlock;
 struct lefMacro;
 struct blockComplex;
 struct CCCap;
+struct R_const;
+struct C_const;
```

Comment:
The PnRDB datatype.h is being shadowed in python. We need to do a few simple things to update that (json reader and writer, new fields in Python).